src/InteractionHandling.tsx

```diff
@@ -105,6 +107,11 @@ export const InteractionHandling = ({
     }
   }, [isAltKeyDown, isShiftKeyDown, isZoomInPossible, isZoomOutPossible, interactionMode])
 
+  const getRubberRange = (anchor: number, position: number): Domain => [
+    Math.min(anchor, position),
+    Math.max(anchor, position),
+  ]
+
   const onMouseDown = (e: React.MouseEvent) => {
     const anchored: Anchored = { variant: 'anchored', anchorX: mousePosition.x }
     if (e.shiftKey) {
```

Comment:
We need to handle one last rubber-band case here, where the shift-click has just happened, but the mouse has not yet been moved. Adding `onZoomInCustomInProgress(...getRubberRange(anchored.anchorX, anchored.anchorX))` after line 118 should fix this.